src/styles.scss

```diff
@@ -30,4 +30,8 @@ body {
 
 .timepicker-backdrop-overlay {
   z-index: 1100 !important;
-}
\ No newline at end of file
+}
+
+.cdk-overlay-container {
+  z-index: 1100 !important;
+}
```

Comment:
I think you could send this in the style of the `details-fields.component.scss` component to avoid its use globally